src/ec/packages/ec-component-language-list/ecl-language-list-splash.html.twig

```diff
@@ -22,8 +22,19 @@
 {% set _items = items|default([]) %}
 {% set _icon_path = icon_path|default('') %}
 {% set _logo = logo|default({}) %}
+{% set _overlay = overlay|default(false) %}
```

Comment:
`overlay`, `close_label`and `title` should be added to parameters description at top of this file

Reply:
It was already missing but `overlay` parameter should also be added on **Notes** tab parameters